docs/DeveloperGuide.adoc

```diff
@@ -729,12 +729,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |see my application's logo |easily identify the application
+
+|`* * *` |user |add a user with a non-alphabet name |add my friend with a non-alphabet name (e.g. Nguyễn)
+
+|`* * *` |user |use command shortcuts |achieve what I want in a shorter time
+
+|`* * *` |user |add and remove tags |
+
+|`* * *` |user |sort my contacts in A-Z order |see my contacts easier
```

Comment:
`see my contacts easier` doesn't sound like a true benefit to the user. Why does the user need to see contacts easily?

Reply:
I'm not sure how should I phrase the justification. All contact app sorts the list by default because it's easier for humans to see in A-Z order.